_episodes/17-responsiblity.md

```diff
@@ -1,74 +1,243 @@
+title: "Using shared resources responsibly"
+teaching: 15
 exercises: 5
 questions:
-- "How can I be a responsible cluster user?"
+- "How can I be a responsible user?"
+- "How can I protect my data?"
+- "How can I best get large amounts of data off an HPC system?"
 objectives:
-- "Be a good person and be nice to other users."
+- "Learn how to be a considerate shared system citizen."
+- "Understand how to protect your critical data."
+- "Appreciate the challenges with transferring large amounts of data off HPC systems."
+- "Understand how to convert many files to a single archive file using tar."
 keypoints:
-- "Don't run stuff on the login node."
+- "Be careful how you use the login node."
+- "Your data on the system is your responsibility."
+- "Plan and test large data transfers."
+- "It is often best to convert many files to a single archive file before transferring."
 - "Again, don't run stuff on the login node."
 - "Don't be a bad person and run stuff on the login node."
 ---
 
-You now have everything you need to run jobs, transfer files, use/install software, and monitor how
-many resources your jobs are using.
+One of the major differences between using remote HPC resources and your own system 
+(e.g. your laptop) is that they are a shared resource. How many users the resource is
+shared between at any one time varies from system to system but it is unlikely you
+will ever be the only user logged into or using such a system.
 
-So here are a couple final words to live by:
+We have already mentioned one of the consequences of this shared nature of the resources:
+the scheduling system where you submit your jobs, but there are other things you need 
+to consider in order to be a considerate HPC citizen, to protect your critical data
+and to transfer data 
 
-## Be kind to the login node
+## Be kind to the login nodes
 
-* The login node is very busy managing lots and lots of jobs! It doesn’t have any extra space to 
-  run computational work. Don’t run jobs on the login node, though quick tasks like downloading 
-  a file or compiling a small program are generally fine. You'll often find limits set on the 
-  login/head node that prevent you from using too many resources.  Attempting to run programs 
-  when resource limits are set will often cause your programs to die with strange errors. 
-  Remember, the login node is to be shared with other users.  
+The login node is often very busy managing lots of users logged in, creating and editing files
+and compiling software! It doesn’t have any extra space to run computational work.
 
-* Most clusters have a queue available for interactive use called 
-  {% for queue in site.workshop_interactive_queues %}{{ queue }}, {% endfor %} etc.  
-  This is generally a better way to run small tests or compile programs.
+Don’t run jobs on the login node (though quick tests are generally fine). A “quick test” is
+generally anything that uses less than 4GB of memory, 4 CPUs, and 10 minutes of time. If you
```

Comment:
`4GB of memory, 4 CPUs, and 10 minutes of time` from my experience, beginners don't know how to measure memory consumption, number of CPUs used may be doable as well as runtime of a process. Therefor it's super tricky to make these kind of recommendations. I'd personally suggest to concentrate on `10 minutes of time` only here. But I am open for suggestions.

Reply:
I agree about the memory point and think you are right: using many cores for 10s will probably not be noticed by other users as it is so transient. I will rephrase and probably reduce to 5 minutes of time.

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/152f118c823ddf914b2c9b09035af1a649f8cc4d